### DISCUSSION TITLE
Win11 preview issues

### DISCUSSION THREAD

**Discussion by DADADADATIAN** [valinet/ExplorerPatcher#1459] [Q&A]
In addition to reinstalling the system, what else can I return to the official version of Win11 (I don't want to delete the data on my computer, my internet speed does not allow me to re-download the data back), and is there any way to make ExplorerPatcher support the build？

**Comment by DADADADATIAN** [comment]
![image](https://github.com/valinet/ExplorerPatcher/assets/115570805/544e9c72-0897-4379-9ab0-ffa32923c0ed)

**Comment by pyrates999** [comment]
Unless it's a dev build, insider versions of windows are not supported because microsoft doesn't provide symbols to download.  More details here: https://github.com/valinet/ExplorerPatcher/wiki/Symbols#how-does-explorerpatcher-obtain-the-necessary-symbol-files

Uninstalling explorer patcher would return your version of windows to what it was.
